tests/tests.js

```diff
@@ -236,6 +237,43 @@ QUnit.test( "useEffect(..)", function test(assert){
 	assert.strictEqual( pActual, pExpected, "call without TNG wrapping context" );
 } );
 
+QUnit.test("useMemo(..)", function test(assert) {
+	function foo(num, rest) {
+		return useMemo(() => {
+			return num;
```

Comment:
These tests don't really *prove* the caching is happening. I'd prefer to use `assert.step(..)` (see other tests) to verify which times the memoized function was invoked vs not. That way we're really sure the memoization is happening.